backpack/extensions/secondorder/hbp/__init__.py

```diff
@@ -115,6 +128,21 @@ def __init__(self, mc_samples=1):
     def get_num_mc_samples(self):
         return self._mc_samples
 
+    def accumulate_backpropagated_quantities(
+        self, existing: Tensor, other: Tensor
+    ) -> Tensor:
+        """Accumulate backpropagated tensors by summation.
+
+
+        Args:
+            existing: first backpropagated tensor.
+            other: second backpropagated tensor.
+
+        Returns:
+            Summation of the two tensors.
+        """
```

Comment:
Remove the docstring and add a `noqa` for `darglint`

Reply:
Attemped to fix this in commit "Fixed doc strings, renamed KFACSumModule to HBPSumModule"
with "# noqa: D102", was there somewhere else I was supposed to make a modification as well?

Reply:
Thanks. Closing.